Validation/SiTrackerPhase2V/src/Phase2ITValidateRecHitBase.cc

```diff
@@ -142,12 +148,17 @@ void Phase2ITValidateRecHitBase::fillRechitHistos(const PSimHit* simhitClosest,
   if (lperr.yy())
     pully = (lp.y() - simlp.y()) / std::sqrt(lperr.yy());
   float eta = geomDetunit->surface().toGlobal(lp).eta();
+  float phi = geomDetunit->surface().toGlobal(lp).phi();
   layerMEs_[key].deltaX->Fill(dx);
   layerMEs_[key].deltaY->Fill(dy);
   layerMEs_[key].pullX->Fill(pullx);
   layerMEs_[key].pullY->Fill(pully);
-  layerMEs_[key].deltaX_eta->Fill(eta, dx);
-  layerMEs_[key].deltaY_eta->Fill(eta, dy);
+
+  layerMEs_[key].deltaX_eta->Fill(std::abs(eta), dx);
```

Comment:
Sorry @sroychow but I realize now that you changed monitoring from eta to abs(eta) but the plots still show eta and no |eta| in the axis legend, so this could be misleading for the future.
WOuld you mind to change the x-axis legend?

Reply:
The same happens in the new resolution plots added

Reply:
updated for both.

Reply:
thx!